dvc/commands/gc.py

```diff
@@ -18,6 +18,7 @@ def run(self):
             all_tags=self.args.all_tags,
             all_commits=self.args.all_commits,
             workspace=self.args.workspace,
+            rev=self.args.rev
```

Comment:
Not sure why we also call `_raise_error_if_all_disabled` given that we also call it inside `self.repo.gc`